gamemode/core/sh_util.lua

```diff
@@ -68,11 +68,65 @@ function nut.util.includeDir(directory, fromLua, recursive)
 	end
 end
 
+-- Credits: framework flux
+-- Function: nut.util.serialize (table toSerialize)
+-- Description: Converts a table into the string format.
+-- Argument: table toSerialize - Table to convert.
+--
+-- Returns: string - pON-encoded table. If pON fails then JSON is returned.
+function nut.util.serialize(tab)
```

Comment:
When would pON encoding actually fail and JSON wouldn't? This seems like a poor way to serialize data because the output seems to be inconsistent. Also, returning empty string with warnings is not a good design because it means some people may call this and not see the error, so it could lead to unexpected behavior.

But, the (de)serialize function should not be included.